.nsprc

```diff
@@ -0,0 +1,4 @@
+{
+  "exceptions": [
```

Comment:
Fwiw, I'm of the opinion we can safely add all three of the current failures as exceptions here:

* Regarding [39](https://npmjs.com/advisories/39) and [48](https://npmjs.com/advisories/39), I don't see any way for uncontrolled or malicious input to reach `uglify-js`.

* Regarding [658](https://npmjs.com/advisories/658), we run on minimum node 8 everywhere and that advisory only applies to node <= 4.

If you want to keep the commit of the script clean that's cool, just add the exceptions to this PR in an extra commit. But we'll need them in this PR because merging is blocked until CI passes!